docs/deployment.md

```diff
@@ -48,7 +48,7 @@ Plug-in packages can directly be installed in *PiWeb Auto Importer* in two diffe
 
 ![Install window](../assets/images/deployment/ai_install_2.png "Install window")
 
-Now the new plug-in is listed in the plug-in management view. However, it is not active yet since a restart is required to actually install and load it. At this point you can use the opportunity to install or remove further plug-ins before finally pressing the `Restart now` button in the info belt at the bottom of the window.
+Now the new plug-in is listed in the plug-in management view. However, it is not active yet since a restart is required to actually install and load it. At this point you continue to install or remove further plug-ins before finally pressing the <span class="nowrap">`Restart now`</span> button in the info belt at the bottom of the window.

```

Comment:
I would like to add "can" in this sentence:
"At this point you can continue to install or remove further plug-ins ..."